WebHostLib/options.py

```diff
@@ -84,8 +84,8 @@ def get_html_doc(option_type: type(Options.Option)) -> str:
                     "max": option.range_end,
                 }
 
-                if issubclass(option, Options.SpecialRange):
-                    game_options[option_name]["type"] = 'special_range'
```

Comment:
since the "new" SpecialRange is a subclass of NamedRange it should still work here. i do agree with hard deprecating it since it's already fixed on source,

Reply:
I believe using SpecialRange from an APWorld will work on both 0.4.3 (as before) and 0.4.4 (emulated) as long as it's frozen or run with -O, so APWorlds should only port to NamedRange once we have a 0.4.4 release or pre-release.